.changeset/tired-places-cough.md

```diff
@@ -0,0 +1,27 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": feature
+---
+
+Provide access to tree content and stored schema without requiring a compatible view schema
+
+Adds `ITreeAlpha` (Which ITree can be down casted to) that exposes access to both the tree content and the schema.
+This allows inspecting the content saved in a shared tree in a generic way that can work on any shared tree.
+
+This can be combined with the existing `generateSchemaFromSimpleSchema` to generate a schema that can be used with [`IIree.viewWith`](https://fluidframework.com/docs/api/fluid-framework/viewabletree-interface#viewwith-methodsignature) to allow constructing a [`TreeView`](https://fluidframework.com/docs/api/fluid-framework/treeview-interface) for any shared tree, regardless of its schema.
+
+Note that the resulting TypeScript typing for such a view will not be friendly: the TreeView APIs are designed for statically known schema. With care and a lot of type casts use is possible, but not recommended when avoidable: see disclaimer on `generateSchemaFromSimpleSchema`.
```

Comment:
```suggestion
Note that the resulting TypeScript typing for such a view will not be friendly: the `TreeView` APIs are designed for statically known schema. Using them is possible with care and a lot of type casts but not recommended if it can be avoided: see disclaimer on `generateSchemaFromSimpleSchema`.
```